### PR TITLE
Integrate signup with GitHub

### LINKED ISSUE #114: integrate-signup-with-github

### PR DESCRIPTION
**Pull request by Remi-dee**
## Task
- [x] Integrate Signup with Github to the app

P2PCoderGif311111

Screenshot 2023-05-31 140709

### CODE REVIEW

File: src/composables/verifySignedIn.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
 import { appAuth } from "./firebaseConfig/config";
-
+const user = appAuth.currentUser;
```
Comment (annonymousauthority) [inline src/composables/verifySignedIn.js:2]: Why change the line?

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/components/signup_comp.js
[the remark this answers is not in the document]
Comment (Remi-dee) [reply src/components/signup_comp.js:1]: i made an update.

File: src/composables/auth_github_signin_popup.js
Comment (annonymousauthority) [inline src/composables/auth_github_signin_popup.js:1]: This function won't work for signup with Github...
Could you check it completely?

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/composables/auth_github_signin_popup.js
Comment (Remi-dee) [reply src/composables/auth_github_signin_popup.js:1]: Alright, will do.

File: src/components/userLogin_comp.js [hunk 2]
```diff
@@ -23,7 +24,10 @@ function UserLoginComp() {
             ></Image>
             Create Account with Google
           </button>
-          <button className="bg-gray-200 flex justify-center items-center p-3 rounded-md w-full shadow-md">
+          <button
+            onClick={signInWithGithub}
```
[the remark this answers is not in the document]
Comment (Remi-dee) [reply src/components/userLogin_comp.js]: For some weird reason, the button doesnt trigger the `onclick` function at times, it rather just reload the page, still trying to figure why that is

File: src/composables/auth_github_signin_popup.js
Comment (Remi-dee) [reply src/composables/auth_github_signin_popup.js:1]: I've updated my request,
The function is working now, after i'd set in all necessary values in the console and app

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**